app/controllers/api/functions.php

```diff
@@ -433,11 +433,8 @@
 
         // Request executor to delete deployment containers
         $ch = \curl_init();
```

Comment:
Just an idea, we have a CURL client in the e2e tests - maybe we can abastract it for this. We are doing alot of duplicate curl stuff all over the place.

Reply:
Yes we need to since there are a couple of places where we're making these calls. We should probably move the Client class to a different namespace for this.